lib/ohai/plugins/aix/virtualization.rb

```diff
@@ -82,8 +82,8 @@
               value = {
                         "mountpoint" => splat[0],
                         "device"     => splat[1],
-                        "vfs"     => splat[2],
-                        "options"     => splat[3].split(","),
+                        "vfs" => splat[2],
+                        "options" => splat[3].split(","),
```

Comment:
i just kinda want to get it in for now.

i suspect there's another cop to turn on that will autofix all of this for us...